Makefile

```diff
@@ -51,7 +51,7 @@ clean:
 
 .PHONY: serve
 serve:
-	$(AT)go run ./cmd/ocm-agent/main.go
+	$(AT)go run ./cmd/ocm-agent/main.go serve --access-token "$TOKEN" --services "$SERVICE" --ocm-url "https://sample.example.com"
```

Comment:
Regarding this, I wonder if we should add this in Makefile after the CLI has env variable or defaults support. For now, is this change required as part of this PR?